src/ml/neural_net/style_transfer/mps_style_transfer_utils.h

```diff
@@ -0,0 +1,88 @@
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h>
+
+@interface TCMPSConvolutionDescriptor:NSObject
+  @property (nonatomic) NSUInteger kernelWidth;
```

Comment:
Nit: ObjC code typically doesn't indent property declarations relative to the interface declaration